### PR TITLE
Add acknowledgements section

### LINKED ISSUE #7: add acknowledgements
From UGM2023 planning TODO list:

- [ ] Add acknowledgements for CZI funding and NF.

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #7 

TODO:
  - Add the CZI EOSS5 grant number & DOI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,3 +51,12 @@ While we will be unable to enforce social distancing guidelines during the UGM,
 Written materials are provided under the [CC-BY-4.0 SA license](LICENSE.md).
 
 The MDAnalysis logo and its derivatives are licensed under the Creative Commons Attribution-NoDerivs 3.0 Unported License.
+
+## Acknowledgements
+
+Please see [`AUTHORS.md`](AUTHORS.md) for a list of contributors to the workshop
+materials.
+
+The MDAnalysis UGM has been made possible in part by CZI grants; DAF2021-237663 (grant DOI https://doi.org/10.37921/426590wiobus), 2022-253062 from the Chan Zuckerberg Initiative DAF, an advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989).
```
Comment (richardjgowers) [inline README.md]: @IAlibay I'm not sure why you've put the first CZI grant?  I've added the grant number for the second, it doesn't have a DOI

Comment (IAlibay) [reply README.md]: Because everyone working on the MDAKits hackathon is paid under EOSS4?

Comment (IAlibay) [reply README.md]: I can remove it if you think it's not necessary though.

Comment (orbeckst) [reply README.md]: I would list EOSS5 first but as Irfan said, EOSS4 is paying for MDAKits stuff and as you remember, we decided to roll the final EOSS4 meeting into the UGM.

**Review by orbeckst — CHANGES_REQUESTED** [review]
Thanks @IAlibay !

We need EOSS5 also in acknowledgments as that’s paying for the “framework”. Please see comment. 

Excuse brevity… typing while waiting for DB.

**Comment by orbeckst** [comment]
(And yes, i saw that EOSS5 is on the todo but it’s important enough to block, just so that it doesn’t slip through.)

**Comment by IAlibay** [comment]
@orbeckst I've switched the order of the grants around, but there's no DOI for EOSS5, I'm not really sure what you want me to do here except from that.

If we want more details about what each funding pot is doing then I'm going to have to defer to someone else.

**Review by orbeckst — APPROVED** [review]
Thanks. I hadn’t realized that BOTH were already listed, I read through the grant numbers too fast.

**Pull request merged**